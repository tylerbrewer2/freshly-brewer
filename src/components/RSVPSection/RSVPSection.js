import React from 'react';

import Section from '../Section';
import Header from '../Header';
import LinkButton from '../LinkButton';

const RSVPSection = () => {
  return(
    <Section centered>
      <Header tag="h1">
        RSVP to our wedding in Baltimore, Maryland on <br />
        December 2nd, 2017.
      </Header>
      <LinkButton to="/rsvp">CONTACT US</LinkButton>
    </Section>
  )
}

export default RSVPSection;