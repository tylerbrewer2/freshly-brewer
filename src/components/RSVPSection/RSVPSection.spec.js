import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import RSVPSection from './RSVPSection';

it('renders correctly', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <RSVPSection />
    </MemoryRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
