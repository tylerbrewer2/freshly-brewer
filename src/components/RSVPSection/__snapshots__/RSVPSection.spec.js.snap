exports[`test renders correctly 1`] = `
<section
  className="sc-bdVaJa cpVWsY">
  <h1
    className="sc-bwzfXH jpWVOo">
    RSVP to our wedding in Baltimore, Maryland on 
    <br />
    December 2nd, 2017.
  </h1>
  <a
    className="sc-bxivhb cKjUkO"
    href="/rsvp"
    onClick={[Function]}>
    CONTACT US
  </a>
</section>
`;
